tests/test_formats.py

```diff
@@ -119,11 +119,16 @@ def test_schemas(self):
                          'keyval': {'public': 'pubkey',
                                     'private': 'privkey'}}),
 
-      'FILEINFO_SCHEMA': (tuf.formats.FILEINFO_SCHEMA,
+      'TARGETS_FILEINFO_SCHEMA': (tuf.formats.TARGETS_FILEINFO_SCHEMA,
                           {'length': 1024,
                            'hashes': {'sha256': 'A4582BCF323BCEF'},
                            'custom': {'type': 'paintjob'}}),
 
+      'METADATA_FILEINFO_SCHEMA': (tuf.formats.METADATA_FILEINFO_SCHEMA,
+                          {'length': 1024,
+                           'hashes': {'sha256': 'A4582BCF323BCEF'},
+                           'version': 1}),
+
```

Comment:
Nit: indentation is not right here

Reply:
I am not sure what you mean by "not right".

Reply:
The dictionary was originally indented to align with the opening delimiter on the line above (PEP 8 style). With the new longer variable name the indentation no longer matches the existing implicit coding style, nor does it match the explicit coding style of the project.

Reply:
Yes, now I get it better.
I was confused that you left the comment in a random location where I placed a new line.